Lesson7/main.js

```diff
@@ -0,0 +1,34 @@
+const ul = document.getElementById("js-ul");
+const loading = document.getElementById("js-loading")
+const fragment = new DocumentFragment();
+const features = [{to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
+{to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}];
+
+const loadingImg = document.createElement('img');
+loadingImg.src = 'loading.gif';
+loading.appendChild(loadingImg)
```

Comment:
IMO
As the number of processes is gradually increasing, I recommend to combine each process into a function.

Reply:
Thank you for your review and feedback Mai-san, 
I have sorted some processes into function as advised.
Kindly check my latest commit b8d018d8d60ded9a6b6c019ca2f061d1184fbeca